tests/test-cases.js

```diff
@@ -694,6 +694,24 @@ var PARSE_TESTS = [
 		}
 	},
 	{
+		description: "Dynamic typing by indices can be determined by function",
+		input: '001,002,003',
+		config: { dynamicTyping: function(field) { return (field % 2) === 0; } },
+		expected: {
+			data: [[1, "002", 3]],
+			errors: []
+		}
+	},
+	{
+		description: "Dynamic typing by headers can be determined by function",
```

Comment:
I don't think it's a duplicate. They are two different scenarios that appear similar. On one hand, the callback takes an integer for parsing without headers. On the other, the callback takes a string for parsing with headers. It doesn't hurt to document those two scenarios in the specs.